Scripts/assignment/emme_assignment.py

```diff
@@ -305,6 +305,7 @@ def print_vehicle_kms(self, resultdata):
                         car_vol -= link[param.link_volumes[ass_class]]
                     kms["car"][vdf] += (param.volume_factors["car"][tp] * car_vol * link.length)
             for line in network.transit_lines():
+                mode = "other"
```

Comment:
Adding mode "other" causes error at line 313, because dictionary `transit_dists` does not have previous value set for this mode. I think operator `+=` requires value to be initialized before assignment.

Reply:
True, I had missed that when staging the changes. I created a new PR to fix it. I think the correct solution would be to use defaultdict here.